docs: hyphenate word third-party (#12967)
renovatebot/renovate: docs/usage/examples/self-hosting.md

npm install -g yarn pnpm
```

> The same goes for any other third-party binary tool like `gradle` or `poetry` - you need to make sure they are installed and the appropriate version before running Renovate.

Author: ```suggestion
The same goes for any other third-party binary tool like `gradle` or `poetry` - you need to make sure it is installed and the appropriate version before running Renovate.
```

I'm not sure the sentence we have now is correct...
We use `they are installed` but the first instance of the noun `tool` is in the singular form. Do you think my suggestion is better grammatically?

Or would this be better?

> The same goes for any other third-party binary **tools** like `gradle` or `poetry` - you need to make sure they are installed and the appropriate version before running Renovate.
